### PR TITLE
chore(package.json) add modules to devDependency

### PR DESCRIPTION
**Pull request by hisabimbola**
#### Summary

add modules required for building this module to devDependency.

#### Reviewers

- [ ] Reviewer 1 has approved the PR
- [ ] Reviewer 2 has approved the PR

### CODE REVIEW

**Comment by hisabimbola** [comment]
@dferber90 The new PR for the devDependency.

I also noticed that babel-eslint was not added to package.json and it's required as the parser in [index.js](https://github.com/commercetools/eslint-config/blob/c42a52dc62ad521e53c7746172606180d900b6fb/index.js#L3).

I am not sure if this was intentional, can be removed Cc @emmenko

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   "homepage": "https://github.com/commercetools/eslint-config",
   "peerDependencies": {
     "eslint": ">=3",
+    "babel-eslint": "^6.1.2",
```
Comment (emmenko) [inline package.json]: Use `>=6`

File: package.json [hunk 2]
```diff
@@ -48,9 +49,15 @@
...
   "devDependencies": {
+    "babel-eslint": "^6.1.2",
     "commitizen": "^2.8.6",
     "cz-conventional-changelog": "^1.2.0",
+    "eslint": ">=3",
+    "eslint-config-airbnb": ">=11",
     "eslint-formatter-pretty": "^1.0.0",
+    "eslint-plugin-import": ">=1",
+    "eslint-plugin-jsx-a11y": ">=2",
+    "eslint-plugin-react": ">=6",
```
Comment (emmenko) [inline package.json]: Use specific versions with caret

File: package.json (hunk 1 above)
Comment (hisabimbola) [inline package.json]: Done here 8871d670

File: package.json (hunk 2 above)
Comment (hisabimbola) [inline package.json]: Done here 8871d67

**Pull request merged**